### DISCUSSION TITLE
Permalink / SEO

### DISCUSSION THREAD

**Discussion by WXQ8NPb5TrbDNqzywcxF** [liedekef/events-made-easy#153] [Q&A]
Hi,

With the permalink option,  is there a way to add a variable,  like the date, or the post id ?
Because I have a lot of event with the same name and I better like to see date instead of incremental number,

**Comment by liedekef** [comment]
sorry, but this is not possible

**Comment by Goliatropin** [comment]
I'm not entirely sure if it's possible to add a variable like the date or post ID with the permalink option. However, there might be other solutions that could work for you. For instance, you could consider using a different naming convention for your events that includes the date, or you could use a tagging system that allows you to easily filter and find specific events. On a related note, have you considered the importance of [SEO](https://www.athinadigital.co.uk/) for your events? It could be a great way to increase your online visibility and attract more attendees. My business recently worked with an [SEO](https://www.athinadigital.co.uk/) agency that helped us improve our online presence and reach a wider audience. They were able to optimize our website and content to make it more search-engine friendly, which resulted in a significant increase in traffic and conversions.
